utility/utils.py

```diff
@@ -713,31 +712,31 @@ def destroy_cluster(cluster_path):
         for volume in volumes:
             aws.detach_and_delete_volume(volume)
 
-        # Remove installer and tarball
+        # Remove installer
         os.remove(installer)
-        os.remove(tarball)
         return TestStatus.PASSED
 
     except Exception:
         log.error(traceback.format_exc())
         return TestStatus.FAILED
 
 
-def download_openshift_installer(version=defaults.INSTALLER_VERSION):
+def get_openshift_installer(version=defaults.INSTALLER_VERSION):
     """
-    Download the openshift installer
+    Get path of the openshift installer binary, download it if not available.
 
     Args:
         version (str): version of the installer to download
 
     Returns:
-        str: name of the installer binary after being unpacked
+        str: path of the installer binary
 
     """
     installer_filename = "openshift-install"
-    tarball = f"{installer_filename}.tar.gz"
-    if os.path.isfile(installer_filename):
+    installer_binary_path = os.path.join(defaults.BIN_DIR, installer_filename)
+    if os.path.isfile(installer_binary_path):
         log.info("Installer exists, skipping download")
+        # TODO: check installer version
```

Comment:
Yeah this seems tricky, probably better to overwrite to be *sure* we got the latest version.

Reply:
Not sure, what would be the best approach here, would need it to another PR.